docs/new-curriculum/js-core-1/week-1/notes.md

```diff
@@ -0,0 +1,349 @@
+
+```js
+"Hello there";
+```
+
+Suppose we want to reuse this string in several different places. For example, we may want to create different greetings for different users, like:
+
+```bash
+"Hello there, Alicia";
+```
+
+```bash
+"Hello there, Barny"
+```
+
+We can use a **variable** to store this string value so it can be referenced and used again.
+
+:::info
+
+A **variable** is a label for a piece of data. We can assign a piece of data to a particular label and then refer back to this label.
+
+:::
+
+We can create a variable in our program by writing a **variable declaration**, in the following way:
+
+```js title="variable declaration"
+const greeting = "Hello there";
+```
+
+We can break down the different syntactic elements of this variable declaration:
+
+- `const` is a keyword used to indicate we're creating a variable.
+- `greeting` is the variable name - like the name of the label for our piece of data.
+- `=` this is the assignment operator. It means assign to the label `greeting` the expression on the right hand side.
+- `"Hello there"` - this is the value we're assigning to the label `greeting`.
+
+We can type this variable declaration into the REPL:
+
+![greeting-variable-declaration](./assets/greeting-variable-declaration.png)
+
+Now we can refer to the label `greeting` again in the REPL:
+
+![reference-greeting](./assets/reference-greeting.png)
+
+Now we have the `greeting` variable stored in memory we can reuse it to build more expressions:
+
+![use-greeting-variable](./assets/use-greeting-variable.png)
+
+:::note
+
+In the example above, we're using backticks to create a template literal. With template literals, we can insert expressions into strings to produce new strings. Any time we want to reference a variable inside a template literal we use a dollar sign and a set of curly braces. The variable name itself is then placed inside the curly braces. The value this variable holds is then placed inside the string.
+
+:::
+
+### Reusing instructions
+
+Let’s consider another scenario. Instead of adding or multiplying numbers, we’ll consider a number like `10.3`.  
+Given the number `10.3`, we could ask:
+
+> 🤔 "What is the nearest whole number to `10.3`?"
+
+The process of finding the nearest whole number to 10.3 is called **rounding**. So we could rephrase the previous question to be:
+
+> 🤔 "What does the number `10.3` **round** to?”
+
+Again we can use Node REPL in order to round the number 10.3 to the nearest whole number. However, in this case, there is no arithmetical operator for rounding the number `10.3` in JavaScript.
+However, we will often want to carry out some kind of operation or series of operations repeatedly. For example, we will often want to round numbers again and again. So in this case we can use a **function**.
+
+:::note
+
+Definition: A function is a reusable set of instructions.
```

Comment:
I do like (and use) the metaphor of instructions often, but would it be worth making it explicit that "instructions" are the same as expressions? (I realise this is not technically correct, since it's also statements).

This helps tie the sections together?